### ISSUE TITLE
Rescuezilla v2.0 ought to create partition.info for Microsoft Reserved partitions

### ISSUE THREAD

**Issue by shasheene** [rescuezilla/rescuezilla#144]
Before releasing Rescuezilla v2.0 I have confirmed that Rescuezilla can make backups of Windows 10 MBR installations and Windows 10 EFI installations and (after zeroing the disk for testing purposes) these backups can be restored using Rescuezilla and Clonezilla. I also confirmed a similar image created by Clonezilla can be restored using Rescuezilla.

However while trying to reproduce a recent issue [reported](https://sourceforge.net/p/rescuezilla/discussion/support/thread/0634ffd596/) by a user, I noticed that Windows 10 MBR backups made by Rescuezilla is missing a file made by Clonezilla for Microsoft Reserved (MSR) partition.

Clonezilla creates a file named eg, sda1.info containing PARTITION_TYPE=Win_boot_reserved", but Rescuezilla v2.0 does not.

This file is only used in Clonezilla's restore process (the `sbin/ocs-expand-gpt-pt` and `sbin/ocs-expand-mbr-pt` scripts) for Clonezilla to determine whether the GPT or MBR partition table is "proportionally restored" when restoring to a larger or smaller disk. From reading the source code, Clonezilla should still makes the same decision even if the file doesn't exist by checking for the MSR's type field.

As stated in the [Limitations Wiki page](https://github.com/rescuezilla/rescuezilla/wiki/Rescuezilla-Limitations), Rescuezilla v2.0 has some support for restoring images to larger (or smaller) disks, but it definitely does not match Clonezilla's proportional restore capabilities yet (which can do advanced things like [resize partitions](https://github.com/rescuezilla/rescuezilla/issues/18) and re-install the GRUB bootloader).

Even without Rescuezilla implementing the proportional restore logic it's important that Rescuezilla create this file so that any image created by Rescuezilla always exactly matches the image created by Clonezilla.

**Comment by shasheene** [comment]
Specifically Rescuezilla should implement the following code (from Clonezilla's `scripts/sbin/ocs-functions`) to create a file named eg, sda1.info containing `PARTITION_TYPE=Win_boot_reserved`.
```
    check_if_windows_boot_reserve_part /dev/$partition
    rc_boot_reserve="$?"
    if [ "$rc_boot_reserve" -eq 0 ]; then
      echo "PARTITION_TYPE=Win_boot_reserved" > $target_dir_fullpath/$partition.info
    fi
```
As described in the previous comment at this stage is not necessary to use this file during a Rescuezilla restore operation.

**Comment by shasheene** [comment]
Fixed in [Rescuezilla v2.1](https://github.com/rescuezilla/rescuezilla/releases/latest).
